docker/.rspec

```diff
@@ -1,4 +1,4 @@
---require pupperware/spec_helper
 --format RspecJunitFormatter
 --out TEST-rspec.xml
 --format documentation
+--fail-fast
```

Comment:
I'm not sure we want `--fail-fast` here? There were a few suites (pe-client-tools-vanagon, puppet-agent, puppet-runtime and r10k) where I didn't add it in my recent sweep, because the suite passing isn't contingent on a few preflight things working first.

Since we only have one test in the suite it also makes no practical difference at the moment.

Reply:
Ok I'll take it off, I don't have an opinion on using it, I just remember you said "we should use --fail-fast everywhere" a few days ago.

Reply:
Yeah, I think my main thinking was catching completely fatal errors in the initial setup and killing the suite as soon as we know. So instead of letting a suite run for 10 minutes in a degraded / unusable state, just let it die.

I wonder if we could also use Ruby `fatal` to do the same thing?